LoopKit/CarbKit/CarbStore.swift

```diff
@@ -42,6 +42,61 @@ public protocol CarbStoreDelegate: class {
 
 }
 
+public protocol CarbStoreProtocol {
```

Comment:
This protocol was added so it's possible to create mock stores to use with LoopDataManager

Reply:
This can be kept in Loop, as well as the conformance extension.

Reply:
@darinkrauss Yes, all of the properties are used. This is the case for all of the protocols; my development process was basically "add the empty protocol to LDM and see what the build errors were"